lstchain/calib/camera/pedestals.py

```diff
@@ -163,6 +164,7 @@ def store_results(self, event):
          event : general event container
         """
 
+        # something wrong if you are here and no statistic is there
         if self.num_events_seen == 0:
             raise ValueError("No pedestal events in statistics, zero results")
```

Comment:
This will exit the execution of the script, simply if there are no interleaved pedestals? 
If so, please transform it into a warning. I am pretty sure we will have otherwise valuable data without interleaved pedestals, without interleaved flatfield events, with none of those, and even with both of them, in some lucky cases.

Reply:
You should not call this function if you do not have collected statistics.  The check must be done before.

Reply:
> You should not call this function if you do not have collected statistics. The check must be done before.

Ok, good, but with your latest fix, is that the case when one runts r0_to_dl1 script?